server/routes/common/reportAProblemPresenter.ts

```diff
@@ -0,0 +1,13 @@
+export default class ReportAProblemPresenter {
+  constructor(private readonly ppUser: boolean) {}
+
+  get contentHTML(): string {
+    if (this.ppUser) {
```

Comment:
Creating HTML is a view responsibility, not a presenter one - I reckon expose the flag on the presenter and let the template do the conditional. (Ideally, the presenter would be able to expose the text, so that it could be tested in a unit test, but that's a bit faffy when you've got those embedded links in there and I'm not sure it's worth trying to solve. Looks like the integration tests have it covered anyway.)

Reply:
Sure, I've moved the logic to the View. The commonController.test should cover the unit test scenario that you mention.